func_adl_xAOD/template/cms/r5/analyzer_cfg.py

```diff
@@ -6,10 +6,16 @@
 
 process.maxEvents = cms.untracked.PSet(input=cms.untracked.int32(-1))
 
+fileNamesArray = []
+with open('/scripts/filelist.txt', 'r') as flist:
+    for line in flist.readlines():
```

Comment:
Should this just be list-comprehension. Something like:

```
fileNamesArray = [f'file:{line}' for line in flist.readlines()]
```
```

Reply:
@gordonwatts would this be better:
```
filelistPath = '/scripts/filelist.txt'
fileNamesArray = [f'file:{line}' for line in open(filelistPath, 'r').readlines()]
fileNamesTuple = tuple(fileNamesArray)
```

Reply:
Or:
```
filelistPath = '/scripts/filelist.txt'
fileNamesTuple = tuple(f'file:{line}' for line in open(filelistPath, 'r').readlines())
```

But now we are down in the weeds. :-)

Reply:
Any of them will do - make the change and then merge, I think!

Reply:
I also added a shebang to remember that the analyzer uses python instead of python3.